contest/contributions/mingione-nowcasting.yml

```diff
@@ -0,0 +1,18 @@
+# Contribution's title, author, abstract
+title: Nowcasting Italian Covid-19 Epidemic
+author: Marco Mingione, Pierfrancesco Alaimo Di Loro # comma-separated
+abstract: |
+  Our work is intended to give people an easy tool for accessing information
+  about the Italian COVID-19 epidemic in an interactive and transparent way.
+  The web application we built is automatically updated every day after the Italian Protezione Civile report (Data Source)
```

Comment:
@minmar94 , did you mean to provide a link to `(Data Source)`? You can use the Markdown syntax
`([Data Source](https://data-source-url))`

Reply:
Yes of course! Should I modify the .yml file directly?

Reply:
[Italian Protezione Civile](https://github.com/pcm-dpc/COVID-19)

Reply:
Is this ok for you?

Reply:
@minmar94, I can take care of it, no worries. I am about to finalize the process soon.